repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/filters/uriNormalization/metrics/single/uri-normalization.cfg.xml

```diff
@@ -42,6 +43,21 @@
                 <parameter name="n" multiplicity="1" case-sensitive="false"/>
             </whitelist>
         </target>
+        -->
+        <target uri-regex="/servers/.*">
+            <whitelist id="pagination-params">
+                <parameter name="a" multiplicity="1" case-sensitive="false"/>
+                <parameter name="r" multiplicity="1" case-sensitive="false"/>
+                <parameter name="n" multiplicity="1" case-sensitive="false"/>
+            </whitelist>
+        </target>
+        <target uri-regex="/resource/.*">
+            <whitelist id="pagination-params">
+                <parameter name="a" multiplicity="1" case-sensitive="false"/>
+                <parameter name="r" multiplicity="1" case-sensitive="false"/>
+                <parameter name="n" multiplicity="1" case-sensitive="false"/>
+            </whitelist>
+        </target>
```

Comment:
The REP-3667 issue is inconsistent, and this temporary fix doesn't really change that.  I recommend removing these lines and expecting an additional test failure until REP-3667 is worked.